front/src/layouts/menu/menu.css

```diff
@@ -0,0 +1,28 @@
+div.main {
+  width: 1440px;
```

Comment:
again?)
menu couldn't be static width. look on fb, github, are they static width?